assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -110,11 +111,10 @@ export function registerDefaults( widgetsAPI ) {
 		priority: 4,
 	}, CONTEXT_PAGE_DASHBOARD );
 
-	widgetsAPI.registerWidget( AREA_PAGE_DASHBOARD_POPULARITY, {
-		title: __( 'Search for individual page or post information', 'google-site-kit' ),
-		style: WIDGET_AREA_STYLES.BOXES,
+	widgetsAPI.registerWidget( WIDGET_URL_SEARCH, {
 		priority: 100,
 		width: WIDGET_WIDTHS.HALF,
 		component: URLSearchWidget,
-	}, CONTEXT_PAGE_DASHBOARD );
+		wrapWidget: false,
+	}, [ AREA_DASHBOARD_POPULARITY, AREA_PAGE_DASHBOARD_POPULARITY ] );
```

Comment:
Oh now I see the confusion here - the ACs were wrong. This should only be available for `AREA_DASHBOARD_POPULARITY`, similarly how it is today. I've fixed that in the ACs, where it was incorrectly referring to `AREA_PAGE_DASHBOARD_POPULARITY`.